docs/content/concepts/partitions-schedules-sensors/partitioned-op-jobs.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: Partitioned op jobs | Dagster
+description: Partitioned op jobs enable launching backfills, where each partition processes a subset of data.
+---
+
+# Partitioned op jobs
+
+<Note>
+  This page is specific to <strong>op jobs</strong>. Looking for
```

Comment:
```suggestion
  This page is specific to <strong>ops</strong>. Looking for
```

following along my suggestion about the "Partitioning ops" title, we could drop "jobs" here.